docker/centos/build.Dockerfile

```diff
@@ -0,0 +1,25 @@
+FROM centos:latest
+
+WORKDIR /build
+
+ADD . /build/parity-ethereum
+
+RUN yum -y update && \
+    yum install -y systemd-devel git make gcc-c++ gcc file binutils && \
+    curl -L "https://cmake.org/files/v3.12/cmake-3.12.0-Linux-x86_64.tar.gz" -o cmake.tar.gz && \
```

Comment:
why not `yum install cmake` ?

Reply:
yum install cmake installs cmake 2.x, but then parity build fails as it needs 3.x, so i decided to install cmake directly. Ideally i would like to get it with yum, but had no luck